src/aeb_fate_generate_ops.erl

```diff
@@ -349,21 +347,22 @@ gen_defines(#{opname := Name, opcode := OpCode}) ->
 
 gen([]) ->
     [];
-gen([{OpName, OpCode, Args, EndBB, Gas, FateFormat, Constructor, Doc} | Rest]) ->
+gen([{OpName, OpCode, EndBB, Gas, FateFormat, Constructor, Doc} | Rest]) ->
+    Args = length(FateFormat),
```

Comment:
Why not use `Arity` instead of `Args` here?

Reply:
I'll fix when I do some other stuff... soon.